src/dndbeyond/content-scripts/character.js

```diff
@@ -848,6 +848,21 @@ async function rollItem(force_display = false, force_to_hit_only = false, force_
             }
         }
 
+        // Handle Dragon Wing * Ranged Weapons
+        if (item_name.includes("Dragon Wing") || item_name === "Flail of Tiamat") {
+            damages.splice(2,damages.length - 2);
+            let possible_damages = damage_types.splice(1,damage_types.length - 1);
+            if (properties.Notes && possible_damages.includes(properties.Notes)) {
+                damage_types.splice(1,0,properties.Notes);
+            }
+            else if (item_name.includes("Dragon Wing")) {
+                damage_types.splice(1,0,"Infused");
+            }
+            else if (item_name === "Flail of Tiamat") {
+                damage_types.splice(1,0,"Chosen Type");
+            }
```

Comment:
This is prone to possible future developer error, if we add a new handled type, but there's no `else` for it, you end up with `damages` array having one more item than `damage_types` array. Having a fallback `else` would prevent that.

Reply:
I think you mean this is prone to ME making errors 😄

Reply:
lol, no! 😁 
It's always a good idea to have an `else` when a situation where all the statements fail could cause issues!

Reply:
We all now what you meant... 😄